fe/src/main/java/org/apache/doris/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -149,6 +165,35 @@ protected RoutineLoadTaskInfo reNewTask(RoutineLoadTaskInfo routineLoadTaskInfo)
         return kafkaTaskInfo;
     }
 
+    public static KafkaRoutineLoadJob fromCreateStmt(CreateRoutineLoadStmt stmt) {
+        // find dbId
```

Comment:
check db semantics here, and you should get database lock before getting table from database

Reply:
If lock of database can get from outside of `Database`